server/libs/userlibs.js

```diff
@@ -119,33 +155,91 @@ const updateUser = function (body) {
     })
 };
 
-const logOut = function(request) {
+const logOut = function(token) {
     return new Promise((resolve,reject) => {
-        let token = request.header('x-authentication');
-        let decoded = {};
-        try {
-            decoded = jwt.verify(token, secretKeys.tokenKey, process.env.JWT_SECRET);
-        } catch (error) {
+        let decoded = decode(token);
```

Comment:
decodeToken should be the name of the function

Reply:
Changed.